### PR TITLE
Add viewport meta tag

### PR DESCRIPTION
**Pull request by Gebon**
@rougier I've added `description` meta tag too, but, I think, its content should be provided by you. Please, tell me what you want to see there.

### CODE REVIEW

**Comment by rougier** [comment]
Thanks. I've no real idea about that.
Maybe the title and a short abstract ? What's the common usage for such metadata ?

**Comment by Gebon** [comment]
[Here](https://moz.com/learn/seo/meta-description) the good description of this meta tag. It's commonly used in search results. There can be a summary of the page. I'd like to see something like This is the book about another approach to the numpy learning, based on experience, practice and descriptive examples. I've never seen other books about numpy, so this is just an example :)
Also, we can omit this meta tag.

**Comment by rougier** [comment]
What about `An open-source book about numpy vectorization techniquers, based on experience, practice and descriptive examples`.

**Comment by Gebon** [comment]
Sounds great!)

**Comment by Gebon** [comment]
Done

File: book.html [hunk 1]
```diff
@@ -26,7 +28,7 @@ <h2 class="subtitle" id="copyright-c-2017-nicolas-p-rougier-nicolas-rougier-inri
 <img alt="data/nc.large.png" src="data/nc.large.png" style="width: 40px;" />
 <div class="line-block">
 <div class="line"><br /></div>
-<div class="line">Latest version - February 2017</div>
+<div class="line">Latest version - Февраль 2017</div>
```
Comment (rougier) [inline book.html]: I guess this means February ?

**Comment by rougier** [comment]
Thanks!

File: book.html (hunk 1 above)
Comment (Gebon) [inline book.html]: Oh, my default locale is ru-RU. I think in rst2html we need to explicitly specify that we want to use en-EN

**Pull request merged**